dev/com.ibm.ws.webcontainer/resources/com/ibm/ws/webcontainer/resources/Messages.nlsprops

```diff
@@ -1614,3 +1614,9 @@ duplicate.samesite.attribute.for.cookie.name.useraction=Check the deployment des
 duplicate.reported.for.samesite.attribute.cookie.name=CWWWC0004W: A {0} cookie name that is configured for the {1} SameSite attribute value was previously identified as a duplicate. No SameSite attribute is added for this cookie.
 duplicate.reported.for.samesite.attribute.cookie.name.explanation=A cookie name cannot be configured for multiple SameSite attribute values. This cookie name has already been identified as a duplicate.
 duplicate.reported.for.samesite.attribute.cookie.name.useraction=Check the deployment descriptor configuration to make sure the cookie name is configured for only one of the following SameSite attribute values: 'SameSiteCookies_Lax', 'SameSiteCookies_None', or 'SameSiteCookies_Strict'. 
+
+# {0} context root that has an invalid ending slash
+# {1} application name
+context.root.can.not.end.with.slash=CWWWC0005W: The {0} context root of the {1} application can not end with a slash (/).
```

Comment:
suggested edit
**change**
The {0} context root of the {1} application can not end with a slash (/).
**To**
The {0} context root of the {1} application cannot end with a slash (/).